docs/user-manual/en/configuring-transports.md

```diff
@@ -295,6 +295,24 @@ The following properties are specific to this native transport:
   a 64bit JVM is detected.  Setting this to `false` will force the use of Java
   NIO instead of epoll. Default is `true`
 
+#### IO_Uring Native Transport
+> **Note:**
+>
+> This is an **INCUBATOR** transport by netty and should be treated as such. 
+> This means it has not had thorough testing at this stage, and users that wish to use this should do their own full testing before using.
+> 
+> We do encourage any feedback on this incubator transport, along with any performance reports positive or negative.
+> 
+> @see https://github.com/netty/netty-incubator-transport-io_uring
+
+On supported platforms IO_Uring can be used @see https://en.wikipedia.org/wiki/Io_uring 
```

Comment:
Supported platforms could be less vague, i.e its Linux (EDIT: which is then mentioned below...so it seems superfluous).

I believe the name is simply io_uring rather than IO_Uring (applies here and most other places IO_Uring is used)